Source/Core/Core/ConfigManager.cpp

```diff
@@ -720,9 +720,8 @@ void SConfig::SetRunningGameMetadata(const DiscIO::IVolume& volume,
                                      const DiscIO::Partition& partition)
 {
   const std::optional<u64> title_id = volume.GetTitleID(partition);
-  if (!title_id)
-    return;
-  SetRunningGameMetadata(volume.GetGameID(partition), *title_id, volume.GetRevision(partition),
+  SetRunningGameMetadata(volume.GetGameID(partition), title_id.value_or(0),
```

Comment:
Sure, changed. This would also look better with GetRevision also returning a std::optional in your PR